lblod/frontend-subsidie-loket
# Feature: make subsidy primary page

## ID

DGS-140

## Description

Make the subsidy index page the primary page, after logging in.

## Type of change


## How to test

Run the frontend, login and verify that the subsidy index page is the primary page after logging in.
